docs/install.md

```diff
@@ -149,10 +149,10 @@ the SGX kernel driver (for Ubuntu 18.04 server):
 ```bash
 apt-get install dkms
 
-DCAP_VERSION=1.2
-UBUNTU_VERSION=ubuntuServer18.04
-DRIVER_REPO=https://download.01.org/intel-sgx/dcap-${DCAP_VERSION}/linux/dcap_installers/${UBUNTU_VERSION}/
-DRIVER_FILE=$(cd /tmp; wget --spider -r --no-parent $DRIVER_REPO 2>&1 | perl  -ne 'if (m|'${DRIVER_REPO}'(.*driver.*)|) { print "$1\n"; }')
+DCAP_VERSION=1.33
```

Comment:
You dropped DCAP for host install, so you might use here also the non-DCAP version? (Using non-DCAP seems fine/right way as the DCAP driver doesn't happen to be really a true super-set of the sgx driver version and we are not doing DCAP yet ...)

Reply:
True. Good to keep things consistent!

Reply:
But also, the rest of this document makes it seem like we _do_ use DCAP everywhere. So if we're going to be consistent, maybe we should remove references to DCAP for the time being?

Reply:
Nothing is using DCAP right now.  At some point i thought DCAP stuff works nicely with both DCAP and EPID. Turns out, though, that while once installed and running, it supports both DCAP and EPID, DCAP stuff requires flexible launch control which exists on more machines than we originally thought but not on all. 

Insofar i would drop all references to dcap (BTW: there is also a still obsolete reference to dcap _1.2_ above the section you changed to 1.3.3 ...) and let's do the same install instruction for PSW here as in docker file?  Seems safest?

Reply:
I see. Yeah, let's make the dockerfile and the install instructions consistent.